pkgs/by-name/yt/yt-dlp/package.nix

```diff
@@ -32,6 +32,7 @@ python3Packages.buildPythonApplication rec {
   dependencies = with python3Packages; [
     brotli
     certifi
+    curl-cffi
```

Comment:
Since this is like 8MB (if cffi is already a dependency elsewhere, which it probably is, then it is only 4MB) it is probably fine to have by default

Reply:
cffi is already brought in by `requests` -> `brotlicffi` as well as `cryptography`, so:

```
$ nvd diff $(nix build nixpkgs#yt-dlp --print-out-paths --no-link) $(nix build .#yt-dlp --print-out-paths --no-link) 
<<< /nix/store/m47znwi2bc09g66j2kn6k7fvfx9cvr38-python3.12-yt-dlp-2024.7.16
>>> /nix/store/15yyji6jvb00ag79sviw756jiqhjrhck-yt-dlp-2024.7.16
Added packages:
[A.]  #1  curl-impersonate-chrome  0.6.1
[A+]  #2  python3.12-curl-cffi     0.7.0b4
[A+]  #3  yt-dlp                   2024.7.16
Removed packages:
[R-]  #1  python3.12-yt-dlp  2024.7.16
Closure size: 94 -> 96 (3 paths added, 1 paths removed, delta +2, disk usage +5.5MiB).
```